assets/app/view/game/map.rb

```diff
@@ -188,6 +191,13 @@ def show_location_names
 
         @show_location_names.nil? ? default : @show_location_names
       end
+
+      def map_zoom
+        stored = Lib::Storage['map_zoom']
+        default = stored.nil? ? 0 : stored
```

Comment:
this is not necessary, you can just do

@map_zoom || Lib::Storage['map_zoom'] || 0

Reply:
Yeah lol
I only did it like this because it's how the others were. I'm changing them all.